intellij/build.gradle

```
@@ -1,15 +1,10 @@
-buildscript {
-  repositories {
-    maven {
-      url "https://plugins.gradle.org/m2/"
-    }
-  }
-  dependencies {
-    classpath "gradle.plugin.org.jetbrains.intellij.plugins:gradle-intellij-plugin:0.4.10"
-  }
-}
+apply plugin: 'saros.gradle.intellij.plugin'
 
-apply plugin: 'org.jetbrains.intellij'
+sarosIntellij {
+  sandboxBaseDir = System.getenv("SAROS_INTELLIJ_SANDBOX") as File
+  localIntellijHome = intellijHome as File
+  intellijVersion = 'IC-2019.2.3'
```

Update to currently used version to avoid the update getting lost on merge?
```suggestion
  intellijVersion = 'IC-2019.2.4'
```